### PR TITLE
[Core][MPI] removing adding to __path__

### PR DESCRIPTION
**Pull request by philbucher**
same as for the other things

### CODE REVIEW

**Comment by philbucher** [comment]
seems like the import of the legacy interface broke
Should we remove it now? Has been around for a while ...

**Comment by roigcarlo** [comment]
If is no longer need I would approve, but seems that structural is still using the legacy in some scripts (see travis)

**Comment by philbucher** [comment]
The import of the legacy mpi interface fails
Thats why I was asking if we should just remove it altogether. Has been deprecated for a long time already

**Comment by roigcarlo** [comment]
+1 For me

**Comment by philbucher** [comment]
done, approval up to you @jcotela

File: applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication_mpi.py [hunk 1]
```diff
@@ -11,8 +11,8 @@
 except ImportError:
     raise Exception("KratosMPI could not be imported!")
 
-if KratosMPI.mpi.size != 2:
-    raise Exception("The MPI tests currently suport only being run with 2 processors!")
+if KratosMultiphysics.DataCommunicator.GetDefault().Size() != 2:
```
Comment (jcotela) [inline applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication_mpi.py]: Very minor, but you could skip a MPI call with 
```suggestion
if KratosMultiphysics.ParallelEnvironment.GetDefaultSize() != 2:
```

**Review by jcotela — DISMISSED** [review]
:+1: for removing this but please try running the python tests in the trilinos application and the mpi core before merging.

**Comment by jcotela** [comment]
BTW, since this is a relatively important deprecation, should we consider adding it in the new Release?

**Comment by roigcarlo** [comment]
Yep, probably a good time. I will add it tomorrow to the 7.1 branch

**Comment by philbucher** [comment]
I ran the tests in the MPI-Core (btw is there no `test_KratosMPICore.py` somewhere?)

Regarding the trilinos I had to check manually because my Trilinos is still broken since the new compile system. Should be fine though, we can update it later in case sth broke

Also regarding the tests in Trilinos, there are many that should be moved to the MPI-Core, since they have nothing to do with Trilinos

**Comment by sunethwarna** [comment]
Hi @philbucher, i pushed changes to RANS, and as well as for one test in Trilinos to change mpi.size call. Could you please have a look?

**Comment by sunethwarna** [comment]
Tests are passing in TrilinosApplication :)

**Comment by philbucher** [comment]
thanks @sunethwarna, looks ot to me, esp if the tests are working :)

@roigcarlo please go ahead

**Comment by philbucher** [comment]
Can someone reapprove pls?

**Review by loumalouomega — APPROVED** [review]

**Comment by jcotela** [comment]
@philbucher regarding your concerns with the test:
- You are right, there is no main file to run all python MPI tests. I was waiting on a standard way to test MPI cases in Python.
- As far as I remember, MPI-only tests in the trilinos application are related to the legacy MPI interface which you are finally killing here. If they still work, it means that the old functionality is still preserved through the new interface, but I do not know if it is really worth to keep them.

**Comment by philbucher** [comment]
@jcotela the remaining tests are for the nodal-neighbors based on GlobalPointers and for the MpiCommunicator, hence they are still needed and should be moved :)

**Pull request merged**